mod-source-record-manager-server/pom.xml

```diff
@@ -232,6 +232,17 @@
       <artifactId>util</artifactId>
       <version>33.0.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.20.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I don't see where you use this dependency.

Reply:
![image](https://user-images.githubusercontent.com/88025830/130409491-5ad8554c-5df2-4db4-b942-ffc47b311421.png)

It's not my changes...